ideas/122-sob-metrics.md

```diff
@@ -0,0 +1,66 @@
+## Preamble
+
+    Idea: 122-sob-metrics
+    Title: Open Bounty - Platform Usage Metrics
+    Status: Draft
+    Created: 2018-04-04
+    Related: 
+
+## Summary
+
+To identify the successful use of a platform, it's important and imperative to be able to measure its usage. As the Open Bounty platform continues to grow and evolve, its important to be able to track its growth, and make data-driven decisions.
+
+One of the key objectives of Open Bounty is to increase the usage of the platform. The purpose of this swarm is to build the tools and means necessary to capture these metrics and be able to report on them on a regular basis.
+
+## Swarm Participants
+
+- Lead Contributor\PM: @arash009 (Arash)
+- Contributor: @siphiuel (Vitaliy)
+- Testing and QA: 
+
+## Product Overview
+
+This swarm will create the mechanism to capture **three** specific metrics on the usage of Open Bounty (specific criteria for each metric yet to be defined):
+
+- **Value transacted** — This metric is to capture the **$USD equivalent** of total tokens that have been paid out in a given timeframe (daily, weekly, monthly, annual)
+- **Active bounty creators** — This metric is to capture the **Monthly Active Contributors** who have posted a minimum of 1 bounty in the previous 30 days.
+- **Active bounty hunters** — This metric is to capture the **Monthly Active Contributors** against each organization who have succesfully completed a bounty.
```

Comment:
Still missing activation metrics IMHO 🙂  see [other comments](https://github.com/status-im/ideas/pull/137#discussion_r179684091) for what I mean by that.

Reply:
I think these are a good start!